nvflare/apis/fl_constant.py

```diff
@@ -222,3 +222,30 @@ class RunProcessKey(object):
     CHILD_PROCESS = "_child_process"
     STATUS = "_status"
     PARTICIPANTS = "_participants"
+
+
+class SystemComponents(object):
+
+    JOB_SCHEDULER = "job_scheduler"
+    JOB_MANAGER = "job_manager"
+    JOB_RUNNER = "job_runner"
+    SERVER_RUNNER = "server_runner"
+    CLIENT_RUNNER = "client_runner"
+    CHECK_RESOURCE_PROCESSOR = "check_resource_processor"
+    CANCEL_RESOURCE_PROCESSOR = "cancel_resource_processor"
+    RESOURCE_MANAGER = "resource_manager"
+    RESOURCE_CONSUMER = "resource_consumer"
+
+
+class WorkspaceConstants:
+    """hard coded file names inside the workspace folder."""
+
+    LOGGING_CONFIG = "log.config"
+    AUDIT_LOG = "audit.log"
+
+    # these two files is used by shell scripts to determine restart / shutdown
+    RESTART_FILE = "restart.fl"
+    SHUTDOWN_FILE = "shutdown.fl"
+
+    WORKSPACE_PREFIX = ""
```

Comment:
is this intended? the workspace_prefix is empty string.

Reply:
This is intended. We could decide to add any prefix to it to only change this one place.